src/test/java/no/sikt/nva/pubchannels/handler/TestChannel.java

```diff
@@ -0,0 +1,109 @@
+package no.sikt.nva.pubchannels.handler;
+
+import static no.sikt.nva.pubchannels.handler.TestUtils.scientificValueToLevel;
+import static no.unit.nva.commons.json.JsonUtils.dtoObjectMapper;
+import static no.unit.nva.testutils.RandomDataGenerator.randomElement;
+import static no.unit.nva.testutils.RandomDataGenerator.randomString;
+import static no.unit.nva.testutils.RandomDataGenerator.randomUri;
+import static nva.commons.core.attempt.Try.attempt;
+import java.net.URI;
+import no.sikt.nva.pubchannels.channelregistry.model.ChannelRegistryJournal;
+import no.sikt.nva.pubchannels.channelregistry.model.ChannelRegistryLevel;
+import no.sikt.nva.pubchannels.channelregistry.model.ChannelRegistryPublisher;
+import no.sikt.nva.pubchannels.channelregistry.model.ChannelRegistrySeries;
+import no.sikt.nva.pubchannels.handler.model.JournalDto;
+import no.sikt.nva.pubchannels.handler.model.PublisherDto;
+import no.sikt.nva.pubchannels.handler.model.SeriesDto;
+import nva.commons.core.paths.UriWrapper;
+
+public class TestChannel {
+
+    private final String identifier;
+    private final Integer year;
+    private final String isbnPrefix;
+    private final ScientificValue scientificValue;
+    private final String discontinued;
+    private final URI sameAs;
+    private final String onlineIssn;
+    private String name;
+    private String printIssn;
+
+    private TestChannel(Integer year, String identifier, String name,
+                        String onlineIssn, String printIssn, String isbnPrefix,
+                        ScientificValue scientificValue,
+                        String discontinued, URI sameAs) {
+
+        this.identifier = identifier;
+        this.year = year;
+        this.name = name;
+        this.onlineIssn = onlineIssn;
+        this.printIssn = printIssn;
+        this.isbnPrefix = isbnPrefix;
+        this.scientificValue = scientificValue;
+        this.discontinued = discontinued;
+        this.sameAs = sameAs;
+    }
+
+    public TestChannel(Integer year, String identifier) {
```

Comment:
This change replaces the call to "randomIssn" with a random string. Could that cause problems? Do we have (or should we have) tests that expect valid ISSNs etc in generated test data?